metadata-models/src/main/pegasus/com/linkedin/dashboard/DashboardInfo.pdl

```diff
@@ -0,0 +1,42 @@
+namespace com.linkedin.dashboard
+
+import com.linkedin.common.AccessLevel
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.ChartUrn
+import com.linkedin.common.Time
+
+/**
+ * Information about a dashboard
+ */
+record DashboardInfo {
+
+  /**
+   * Title of the dashboard
+   */
+  title: string
+
+  /**
+   * Detailed description about the dashboard
+   */
+  description: string
+
+  /**
+   * Charts in a dashboard
+   */
+  charts: array[ChartUrn] = [ ]
+
+  /**
+   * Captures information about who created/last modified/deleted this dashboard and when
+   */
+  lastModified: ChangeAuditStamps
```

Comment:
same as the previous comment. Why do we need to store audit stamp in the aspect?

Reply:
Same answer.